hotosm/tasking-manager
# Adds ml-enabler integration

This adds ml-enabler integration to TM. (continued from #1787 )

To test:

Update requirements

run looking-glass locally or update environment variables to point to a looking glass instance (see config.py)

Create a project on step 3 you'll see the "ML Layer" button, click it to generate a prediction, this will take a while;
![Screen Shot 2019-07-24 at 11 06 38 PM](https://user-images.githubusercontent.com/9713/61885955-01df3200-aebc-11e9-8161-b0975705c502.png)

After the prediction finishes it should look like this:

![Screen Shot 2019-07-24 at 11 09 19 PM](https://user-images.githubusercontent.com/9713/61885961-03a8f580-aebc-11e9-88ad-a73fd7f7f0af.png)


After that the project can be created and task annotations will be created too so project detail will have the "ML Layer" button on too.
